CMakeLists.txt

```diff
@@ -71,6 +74,34 @@ else()
   message(STATUS "OpenCL Not Found")
 endif()
 
+if(DPCPP_FOUND)
+  find_program(DPCT_FOUND dpct QUIET)
+  if(DPCT_FOUND)
+    message(STATUS "Found DPC++: " ${DPCPP_FOUND})
+    message(STATUS "Found DPCT: " ${DPCT_FOUND})
+    set(CUDA_INCLUDE_PATH $ENV{CUDA_HOME}/include)
```

Comment:
Can we use what cmake found instead of `$ENV{CUDA_HOME}`?

Here is my suggestion:

```
if(DPCT_FOUND)
  ... 
  if(CMAKE_CUDA_COMPILER)
  ... 
     add_custom_command(
        ...
        COMMAND dpct ... --cuda-include-path=${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}

  else()
     message(STATUS "Found DPCT, but CUDA NOTFOUND"
...
```

Reply:
Done